apps/avifenc.c

```diff
@@ -653,7 +653,7 @@ typedef struct
     int keyframeInterval;
     avifBool ignoreExif;
     avifBool ignoreXMP;
-    avifBool ignoreICC;
+    avifBool ignoreProfile;
```

Comment:
Nit: In local variables and parameters, I think it is fine to be more verbose and use the name "ignoreColorProfile".

The command-line option "--ignore-profile" can be short because we may need to type it manually.